File: my-integration-test-timeline/present/src/test/java/br/com/leonardoferreira/present/FakerConfiguration.java
package br.com.leonardoferreira.present;

import com.github.javafaker.Faker;
import java.util.Random;
import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;

@Configuration
public class FakerConfiguration {

    @Bean
    public Faker fakerBean() {
        long seed = System.currentTimeMillis();
        System.out.println(seed);
        Random random = new Random(seed);
        return new Faker(random);
    }

}

File: my-integration-test-timeline/future/src/test/kotlin/br/com/leonardoferreira/future/configuration/FakerConfiguration.kt
package br.com.leonardoferreira.future.configuration

import com.github.javafaker.Faker
import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration

@Configuration
class FakerConfiguration {

    @Bean
    fun faker(): Faker {
        return Faker()
    }

}
